phoenix-core-client/src/main/java/org/apache/phoenix/iterate/OrderedResultIterator.java

```diff
@@ -151,6 +159,14 @@ public Expression apply(OrderByExpression column) {
     private Tuple dummyTuple = null;
     private long byteSize;
     private long pageSizeMs;
+    private Scan scan;
+    private byte[] scanStartRowKey;
+    private byte[] prevScanStartRowKey;
+    private Boolean prevScanIncludeStartRowKey;
```

Comment:
nit: boolean? (small b?)

Reply:
We need Boolean here because only server side execution of this iterator sets this value so we have check in place to see if the value is not null.
e.g.
```
            if (firstScan && serverSideIterator && prevScanStartRowKey != null &&
                    prevScanIncludeStartRowKey != null) {
```